packages/web-components/fast-foundation/src/tabs/tabs.ts

```diff
@@ -364,6 +364,22 @@ export class FASTTabs extends FASTElement {
             this.tabs.length - 1,
             this.activeTabIndex + adjustment
         );
+
+        if (this.activeTabIndex > this.prevActiveTabIndex) {
+            while (
```

Comment:
Can't these "while" loops get out of thre range of valid indexes if there are no focusable tabs found as you increment/decrement?

Reply:
Ok, nvm, I see it now.